base/bytes.h

```diff
@@ -32,7 +32,7 @@ template <
 		!std::is_const_v<Container>
```

Comment:
It seems to me @john-preston meant to check here const qualifier of an element, not constancy of the container itself. Something like `std::is_const_v<typename Container::value_type>`.